ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/KsqlRestConfigTest.java

```diff
@@ -40,38 +41,27 @@
   }
 
   @Test
-  public void testGetKsqlStreamsProperties() {
-    final long BASE_COMMIT_INTERVAL_MS = 1000;
-    final long OVERRIDE_COMMIT_INTERVAL_MS = 100;
-
-    final String OVERRIDE_BOOTSTRAP_SERVERS = "ksql.io.confluent:9098";
-
-    assertNotEquals(BASE_COMMIT_INTERVAL_MS, OVERRIDE_COMMIT_INTERVAL_MS);
-
+  public void testGetKsqlConfigProperties() {
     Map<String, Object> inputProperties = getBaseProperties();
-    inputProperties.put(
-        StreamsConfig.COMMIT_INTERVAL_MS_CONFIG,
-        BASE_COMMIT_INTERVAL_MS
-    );
-    inputProperties.put(
-        KsqlRestConfig.KSQL_STREAMS_PREFIX + StreamsConfig.COMMIT_INTERVAL_MS_CONFIG,
-        OVERRIDE_COMMIT_INTERVAL_MS
-    );
-    inputProperties.put(
-        KsqlRestConfig.KSQL_STREAMS_PREFIX + StreamsConfig.BOOTSTRAP_SERVERS_CONFIG,
-        OVERRIDE_BOOTSTRAP_SERVERS
-    );
+    inputProperties.put(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "earliest");
+    inputProperties.put(KsqlConfig.KSQL_SERVICE_ID_CONFIG, "test");
 
-    Map<String, Object> testProperties = new KsqlRestConfig(inputProperties).getKsqlStreamsProperties();
+    KsqlRestConfig config = new KsqlRestConfig(inputProperties);
 
-    assertEquals(
-        OVERRIDE_COMMIT_INTERVAL_MS,
-        testProperties.get(StreamsConfig.COMMIT_INTERVAL_MS_CONFIG)
-    );
-    assertEquals(
-        OVERRIDE_BOOTSTRAP_SERVERS,
-        testProperties.get(StreamsConfig.BOOTSTRAP_SERVERS_CONFIG)
-    );
+    Map<String, Object> ksqlConfigProperties = config.getKsqlConfigProperties();
+    assertThat(ksqlConfigProperties.size(), equalTo(6));
```

Comment:
alternatively, you could create a `Map` with all the expected properties and do `assertThat(ksqlConfigProperties, equalTo(expectedProperties))`